Trivia and prisoner dillema logo

Dadik11 opened the pull request:
Change filenames idk
did not integrate them anywhere, just plain files
👍

Dadik11 committed:
Trivia and prisoner dillema logo

joseph-gerald commented:
Make sure it contrasts with the game wheel (talking about the light themed prisoners dilemma) try implementing it into the game wheel by going into [game array](https://github.com/joseph-gerald/party-game/blob/032ca81e38c314d659091712905f8137fd612b6a/src/handlers/game.js#L16), and then creating and starting a game by yourself to see how it looks like with the wheel. Here's a [codespace instance](https://fictional-umbrella-6wj7qv7x756cg75-3000.app.github.dev/) with the prisoner dilemma on the wheel, looks cursed, make it dark themed if possible.

Currently it skips tons of animations when on local host should make that an environmental variable instead...

*ps: try to use [gitmoji](https://gitmoji.dev/) makes it easier to know what a commit contains at a glance)*

Dadik11 committed:
Merge branch 'joseph-gerald:main' into patch-1

Dadik11 committed:
SkipAnimation & logos for trivia and prisonners dillema 🍱 ✨

Dadik11 commented:
Changed the logo for trivia to yours, just drawn differently.
Implemented everything, looks just OK *for now*. When ~3 more games are added you the logos will be small 🤌 anyway 🤷‍♂️ then I could update their dimensions if look cursed.

> Currently it skips tons of animations when on local host should make that an environmental variable instead...

made it so you can add `?skipAnimations=1` to the url (only if you're on localhost)

Here's how it looks:
![wheel](https://github.com/joseph-gerald/party-game/assets/98746815/3c890f8f-2288-44b7-8d54-f801aa028b6e)

joseph-gerald commented:
Great 👍
